src/Features/VisualBasic/Portable/Diagnostics/Analyzers/VisualBasicQualifyMemberAccessDiagnosticAnalyzer.vb

```diff
@@ -0,0 +1,33 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Collections.Immutable
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.Diagnostics.QualifyMemberAccess
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.CodeFixes.QualifyMemberAccess
+
+    <DiagnosticAnalyzer(LanguageNames.VisualBasic)>
+    Friend NotInheritable Class VisualBasicQualifyMemberAccessDiagnosticAnalyzer
+        Inherits QualifyMemberAccessDiagnosticAnalyzerBase(Of SyntaxKind)
+
+        Private Shared ReadOnly s_kindsOfInterest As ImmutableArray(Of SyntaxKind) = ImmutableArray.Create(SyntaxKind.IdentifierName)
+
+        Protected Overrides Function GetLanguageName() As String
+            Return LanguageNames.VisualBasic
+        End Function
+
+        Protected Overrides Function GetSupportedSyntaxKinds() As ImmutableArray(Of SyntaxKind)
+            Return s_kindsOfInterest
+        End Function
+
+        Protected Overrides Function IsCandidate(node As SyntaxNode) As Boolean
+            If TypeOf node.Parent Is MemberAccessExpressionSyntax Then
+                Return If(CType(node.Parent, MemberAccessExpressionSyntax)?.Expression?.Kind(), SyntaxKind.None) <> SyntaxKind.MeExpression
```

Comment:
I agree.  However, what i meant was: can we have tests to make sure that we don't mess up in cases where we have 'base' or 'static' or 'pointer' access :)

Right now you're checking just for MemberAccessExpression.  That encompasses a lot.  I want to make sure we don't mess things up when the above returns 'true' when called on 'MyBase.Blah'.